GUI/SettingsDialog.resx

```diff
@@ -160,9 +160,10 @@
   <data name="RefreshPostLabel.Text" xml:space="preserve"><value>minute(s)</value></data>
   <data name="PauseRefreshCheckBox.Text" xml:space="preserve"><value>Pause refreshing</value></data>
   <data name="MoreSettingsGroupBox.Text" xml:space="preserve"><value>More Settings</value></data>
-  <data name="AutoSortUpdateCheckBox.Text" xml:space="preserve"><value>Automatically sort by "Update"-column when clicking "Add available updates"</value></data>
+  <data name="LanguageSelectionLabel.Text" xml:space="preserve"><value>Language (Requires Restart):</value></data>
```

Comment:
```suggestion
  <data name="LanguageSelectionLabel.Text" xml:space="preserve"><value>Language (requires restart):</value></data>
```

The parenthesized part doesn't need capitalization in English

Reply:
I did it because all the other occurrences were capitalized too.
I can change all of them :+1:

Reply:
Hmm, I don't think anything else needs to change... :worried:

Reply:
https://github.com/DasSkelett/CKAN/blob/2506ce06a09a73bee419bf563c53d7779561170b/GUI/SettingsDialog.resx#L165-L166
What about those?

Reply:
Oh! Yes, change those as well. :relieved: